src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,71 @@
+package calculator;
+
+public class StringCalculator {
+
+    private static final String SUPPORT_OPERATOR = "+-*/";
+    private static final String DELIMITER = " ";
+
+    public int calculate(String input) {
+        validateInput(input);
+
+        String[] splitInput = input.split(" ");
```

Comment:
선언하신 상수를 이용하여 치환해보면 어떨까요?
```suggestion
    private static final String DELIMITER = " ";

    public int calculate(String input) {
        validateInput(input);
        String[] splitInput = input.split(DELIMITER);
```

Reply:
앗..그렇네요 여기는 변경하지 않았네요
297736db3d4ef41b3a14faa6d9544c57e8bb1a35
커밋에서 수정하였습니다 !